### PR TITLE
Update to Chromium 86.0.4240.183

### PR DESCRIPTION
**Pull request by tangalbert919**
Notable changes:
- A snapshot of Clang 12 is used.

Current issues:
- ~~v8 is still preventing a successful build.~~ This issue has been fixed.

### CODE REVIEW

**Comment by Luka-Filipovic** [comment]
This build should pass if combined with #60 and without `windows-fix-v8_build` patch.

**Comment by tangalbert919** [comment]
I was able to get a successful build. This should be safe to merge now.

File: patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ 
+ def parse_options():
+     parser = optparse.OptionParser(usage="%prog [options] TASK...")
+@@ -53,6 +56,10 @@ def parse_options():
+ 
+ 
+ def main():
++    # This fix build issue on Windows caused by missing google dependencies.
++    if os.name == 'nt':
++        shutil.copy('../../third_party/llvm-build/Release+Asserts/bin/clang-format.exe', '../../buildtools/win')
```
Comment (Eloston) [inline patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch:20]: @AKile9V Doesn't this copy the binary every time this program is invoked? Can we check for existence before copying?

**Review by Eloston — CHANGES_REQUESTED** [review]
LGTM, just a question about the v8 build fix

File: patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch (hunk 1 above)
Comment (AKile9V) [reply patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch:20]: Sure, I think that is good idea. I'll try to add some checker if clang-format is already there, test it on my build and then will fill in

File: patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch [hunk 2]
```diff
@@ -11,13 +11,14 @@
...
      parser = optparse.OptionParser(usage="%prog [options] TASK...")
-@@ -53,6 +56,10 @@ def parse_options():
+@@ -53,6 +57,11 @@ def parse_options():
  
  
  def main():
 +    # This fix build issue on Windows caused by missing google dependencies.
 +    if os.name == 'nt':
-+        shutil.copy('../../third_party/llvm-build/Release+Asserts/bin/clang-format.exe', '../../buildtools/win')
++        if not os.path.exists('../../buildtools/win/clang-format.exe')
```
Comment (Eloston) [inline patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch:20]: Is this even valid syntax without the colon `:` at the end?

**Review by Eloston — CHANGES_REQUESTED** [review]

File: patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch (hunk 2 above)
Comment (tangalbert919) [reply patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch:20]: Oh right

Comment (tangalbert919) [reply patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch]: This is fixed now

**Review by Eloston — APPROVED** [review]
Thanks :+1:

**Pull request merged**